diff --git a/CMakeLists.txt b/CMakeLists.txt
index e9414d9..2bc72a8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,9 +33,9 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(GLIB REQUIRED IMPORTED_TARGET glib-2.0)
 
 # find a bunch of packages (vcpkg provides CONFIG mode packages)
 find_package(rtprocess CONFIG REQUIRED)
 find_package(exiv2 CONFIG REQUIRED)
-find_package(libraw CONFIG REQUIRED)
+find_package(libraw REQUIRED)
 find_package(JPEG REQUIRED)
 find_package(TIFF REQUIRED)
 find_package(OpenMP REQUIRED)
@@ -205,8 +205,8 @@ target_link_libraries(filmulator
         Qt5::Widgets
         Qt5::Quick
         Qt5::Qml
         Exiv2::exiv2lib       # exiv2 modern target
-        libraw::raw_r         # libraw modern target
+        ${LIBRAW_LIBRARIES}
         JPEG::JPEG            # CMake built-in JPEG target
         TIFF::TIFF            # TIFF modern target
         ${LENSFUN_TARGET}
